frontend/src/components/CourseCard/index.tsx

```diff
@@ -17,11 +17,12 @@ export interface ICourseCardProps {
   level: string;
   rating: number;
   onOpenClick: () => void;
+  hideButton?: boolean;
 }
 
 export const CourseCard: React.FunctionComponent<ICourseCardProps> = ({
   category, imageSrc, name, author, duration,
-  level, rating, onOpenClick
+  level, rating, onOpenClick, hideButton
```

Comment:
Put them in a column:
category,
imageSrc,
name,
author,
duration,
  level, rating, onOpenClick	  level, rating, onOpenClick, hideButton